modules/plugins/languages/beancount.nix

```
@@ -0,0 +1,122 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}: let
+  inherit (builtins) attrNames;
+  inherit (lib.options) mkOption mkEnableOption literalExpression;
+  inherit (lib.meta) getExe';
+  inherit (lib.modules) mkIf mkMerge;
+  inherit (lib.lists) isList;
+  inherit (lib.types) anything attrsOf either enum listOf package str;
+  inherit (lib.nvim.types) mkGrammarOption;
+
+  cfg = config.vim.languages.beancount;
+
+  defaultServer = "beancount-language-server";
+  servers = {
+    beancount-language-server = {
+      rootmarkers = [".git"];
+      filetypes = ["beancount" "bean"];
+      init_options = cfg.lsp.initOptions;
+      cmd =
+        if (isList cfg.lsp.package)
+        then cfg.lsp.package
+        else [(getExe' cfg.lsp.package cfg.lsp.server)];
```

```suggestion
      cmd = [(getExe' cfg.lsp.package cfg.lsp.server)];
```
same as above, customizations should be done via `vim.lsp.servers`